pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "AD_Miner"
 version = "0.2.1"
 description = "AD Miner is an Active Directory audit tool that leverages cypher queries to crunch data from the Bloodhound graph database to uncover security weaknesses."
-authors = [ "Thomas Girard", "Jean-Michel Besnard", "Alexandre Lac" ]
+authors = [ "Thomas Girard", "Jean-Michel Besnard", "Alexandre Lac" , "Andreu Jonathan" ]
```

Comment:
Should be this :
authors = [ "Jean-Michel Besnard", "Jonathan Andreu" , "Joseph Pradines", "Emilien Vannier", "Samy Bontemps", "Thomas Girard", "Tanguy Boisset", "Vangelis Hoareau", "Clément Teytaud", "Lucas Bozon", "Alexandre Lac" ]